services/mailer/mail_team_invite.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package mailer
+
+import (
+	"bytes"
+	"context"
+
+	org_model "code.gitea.io/gitea/models/organization"
+	user_model "code.gitea.io/gitea/models/user"
+	"code.gitea.io/gitea/modules/base"
+	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/setting"
+	"code.gitea.io/gitea/modules/templates"
+	"code.gitea.io/gitea/modules/translation"
+)
+
+const (
+	tplTeamInviteMail base.TplName = "team_invite"
```

Comment:
we might aso mention this template in https://docs.gitea.io/en-us/mail-templates/ (same goes for "release"

Reply:
The docs page does not contain a list of templates, only information about the mail templates?

Reply:
yes but we might wana add the list ...

Reply:
optional plus-points ;)

Reply:
#21509